package.json

```diff
@@ -29,7 +29,9 @@
     "@skyux/i18n": "^4.0.0",
     "@skyux/indicators": "^4.0.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@blackbaud/skyux-builder-plugin-code-block": "1.1.0"
```

Comment:
☝️ This should be a devDependecy.

Reply:
Nice catch! Thank you for this! I made the fix so hopefully we're good to go now 🤞